include/drivers/interrupt_controller/gic.h

```diff
@@ -219,6 +210,47 @@
 
 #endif /* CONFIG_GIC_VER <= 2 */
 
+/* GICD_ICFGR */
+#define GICD_ICFGR_MASK			BIT_MASK(2)
+#define GICD_ICFGR_TYPE			BIT(1)
+
+/*
+ * Common Helper Constants
+ */
+/* GICD_TYPER.ITLinesNumber 0:4 */
+#define IT_LINES_NO_MASK		0x1f
```

Comment:
Still the same issue. If these are going to live in a public header, these should be `GIC_` prefixed.

Also can we make this name align with the existing nomenclature (i.e. something like `GICD_TYPER_ITLINESNUM_MASK`).

Reply:
ACK